run_tests.sh

```diff
@@ -1,3 +0,0 @@
-#! /bin/sh
-
-nosetests --verbose --with-doctest --with-coverage --cover-package pydocx $@ && find pydocx -name '*.py' | xargs flake8
```

Comment:
How to do you feel about moving this command to a `Makefile`? It's slightly more standard than this approach. You would type `make test` to run the tests. Thoughts?